frontend/src/components/Login/Input.tsx

```diff
@@ -0,0 +1,49 @@
+import { makeStyles, TextField } from "@material-ui/core";
+import React from "react";
+
+const useStyles = makeStyles({
```

Comment:
Do you just use material-ui because you can't style some component on your own? Shame...